config/locales/en/brexit_landing_page.yml

```diff
@@ -1,49 +1,29 @@
 en:
```

Comment:
re-ordered a lot of these to match the page, tried to remove ones i couldn't see being used anymore and renamed a bunch to match content designers jargon